src/utilities/__tests__/printSchema-test.ts

```diff
@@ -663,7 +680,7 @@ describe('Type System Printer', () => {
       schema {
         query: Query
       }
-      
+
```

Comment:
Thanks for spotting it!
I applied global fix, here: https://github.com/graphql/graphql-js/pull/3970